src/main/java/com/stripe/exception/CardException.java

```diff
@@ -5,7 +5,15 @@
 
 	private String code;
 	private String param;
+	private String declineCode;
 
+	public CardException(String message, String code, String param, String declineCode, Throwable e) {
+		super(message, e);
+		this.code = code;
+		this.param = param;
+		this.declineCode = declineCode;
+	}
+	
 	public CardException(String message, String code, String param, Throwable e) {
```

Comment:
I don't see any other call-sites for this constructor, so I think it can go away / just be replaced by your new constructor!